docs/src/pages/docs/Connecting to Databases/ui.mdx

```diff
@@ -0,0 +1,77 @@
+---
+name: Using Database Connection UI
+menu: Connecting to Databases
+route: /docs/databases/db-connection-ui
+index: 1
+version: 1
+---
+
+Here is the documentation on how to leverage the new DB Connection UI. This will provide admins the ability to enhance the UX for users who want to connect to new databases.
+
+**Add Giph for before and after**
+
+There are now 3 steps when creating a new dynamic form:
```

Comment:
another nit.. I'd recommend: There are now 3 steps when connecting to a database in the new UI.